site/content/en/docs/Tasks/namespaces_and_names.md

```diff
@@ -6,3 +6,114 @@ date: 2023-10-14
 description: >
   Working with Namespaces and Names
 ---
+
+The Namespace can be set for all Resources in a project by adding the `namespace` entry to the `kustomization.yaml` file. Consistent naming conventions can be applied to Resource Names in a project with the `namePrefix` and `nameSuffix` fields.
+
+## Add Namespace
+The Namespace for all namespaced Resources in a project can be set with the `namespace` field. This will override Namespace values that already exist.
```

Comment:
nit: Can we add the sentence about "generated Resources (e.g. ConfigMaps and Secrets)" in the following paragraph and maybe add to the existing example? https://kubectl.docs.kubernetes.io/guides/config_management/namespaces_names/#setting-namespace

I actually found this detail very neat. Never really thought about it before :)